cpp/src/io/orc/reader_impl.hpp

```diff
@@ -223,6 +223,7 @@ class reader::impl {
   bool _use_np_dtypes = true;
   std::vector<std::string> _decimal_cols_as_float;
   std::vector<std::string> decimal128_columns;
+  bool is_decimal128_enabled;
   data_type _timestamp_type{type_id::EMPTY};
   reader_column_meta _col_meta;
```

Comment:
Inconsistency in what is initialized and how things are initialized here. Two `bool` members above are initialized to true using `=`. This `bool` is not initialized. `data_type` is initialized using `{}`. Suggest adding initialization for everything and doing them all the same way.

Reply:
Done, only left the vectors without "visible" initialization.